_ml-commons-plugin/gpu-acceleration.md

```diff
@@ -0,0 +1,304 @@
+---
+layout: default
+title: GPU acceleration
+parent: Model-serving framework
+nav_order: 150
+---
+
+# GPU acceleration 
+
+When running a natural language processing (NLP) model in your OpenSearch cluster with a machine learning (ML) node, you can achieve better performance on the ML node using graphics processing unit (GPU) acceleration. GPUs can work in tandem with the CPU of your cluster to speed up the model upload and training. 
+
+## Supported GPUs
+
+Currently, ML nodes support any CUDA-enabled GPUs, including the following:
+
+- [NVIDIA G-series instances](https://aws.amazon.com/nvidia/)
+- [AWS Inferentia](https://aws.amazon.com/machine-learning/inferentia/)
+
+If you need GPU power, you can provision GPU instances through [Amazon Elastic Compute Cloud (Amazon EC2)](https://aws.amazon.com/ec2/). For more information on how to provision a GPU instance, see [Recommended GPU Instances](https://docs.aws.amazon.com/dlami/latest/devguide/gpu.html).
+
+## Supported images
+
+You can use GPU acceleration with both [Docker images](https://github.com/NVIDIA/nvidia-docker) and [Amazon Machine Images (AMIs)](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html).
```

Comment:
We support cuda version 11.6, so we support nvidia/cuda docker with 11.6 version. We tested with docker `nvidia/cuda:11.6.2-cudnn8-devel-ubuntu20.04`